controller.go

```diff
@@ -372,6 +372,12 @@ func syncPV(pv *PV) {
 				}
 			} else {
 				// The PV must have been created with this ptr; leave it alone.
```

Comment:
In my interpretation, `// leave it alone` means that we won't bind it and we won't remove the "pre-binding" either. We just update the status.

And about `Available` state - that's a good question. `types.go` say:
```
	// used for PersistentVolumes that are not available
	VolumePending PersistentVolumePhase = "Pending"
	// used for PersistentVolumes that are not yet bound
	// Available volumes are held by the binder and matched to PersistentVolumeClaims
	VolumeAvailable PersistentVolumePhase = "Available"
```

Here I would vote for `Available` as "not yet bound".